lib/api/response.js

```diff
@@ -2,8 +2,8 @@ const Request = require('./request');
 const { convertKeysToCamelCase } = require('../utils');
 const { ZendeskRequestError } = require('./errors');
 
-function Response(response = {}, { authHeader } = {}) {
-  const body = convertKeysToCamelCase(response.body);
+function Response(response = {}, { authHeader, parseBodyManuallyAsJson = false } = {}) {
+  const body = convertKeysToCamelCase(parseBodyManuallyAsJson ? JSON.parse(response.body) : response.body);
```

Comment:
yowser... Not sure I'm entirely happy with this... Please can we inspect the content-type of the response instead?